tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -428,49 +419,37 @@ def targets():
             PythonArtifact(
                 "manylinux2014", "aarch64", "cp312-cp312", presubmit=True
             ),
-            PythonArtifact(
-                "linux_extra", "armv7", "cp37-cp37m", presubmit=True
-            ),
-            PythonArtifact("linux_extra", "armv7", "cp38-cp38"),
+            PythonArtifact("linux_extra", "armv7", "cp38-cp38", presubmit=True),
             PythonArtifact("linux_extra", "armv7", "cp39-cp39"),
             PythonArtifact("linux_extra", "armv7", "cp310-cp310"),
             PythonArtifact("linux_extra", "armv7", "cp311-cp311"),
             PythonArtifact(
                 "linux_extra", "armv7", "cp312-cp312", presubmit=True
             ),
+            PythonArtifact("musllinux_1_1", "x64", "cp38-cp38", presubmit=True),
+            PythonArtifact("musllinux_1_1", "x64", "cp39-cp39"),
             PythonArtifact("musllinux_1_1", "x64", "cp310-cp310"),
             PythonArtifact("musllinux_1_1", "x64", "cp311-cp311"),
             PythonArtifact(
                 "musllinux_1_1", "x64", "cp312-cp312", presubmit=True
             ),
-            PythonArtifact(
-                "musllinux_1_1", "x64", "cp37-cp37m", presubmit=True
-            ),
-            PythonArtifact("musllinux_1_1", "x64", "cp38-cp38"),
-            PythonArtifact("musllinux_1_1", "x64", "cp39-cp39"),
+            PythonArtifact("musllinux_1_1", "x86", "cp38-cp38", presubmit=True),
+            PythonArtifact("musllinux_1_1", "x86", "cp39-cp39"),
             PythonArtifact("musllinux_1_1", "x86", "cp310-cp310"),
             PythonArtifact("musllinux_1_1", "x86", "cp311-cp311"),
             PythonArtifact(
                 "musllinux_1_1", "x86", "cp312-cp312", presubmit=True
             ),
-            PythonArtifact(
-                "musllinux_1_1", "x86", "cp37-cp37m", presubmit=True
-            ),
-            PythonArtifact("musllinux_1_1", "x86", "cp38-cp38"),
```

Comment:
This is removing several 3.8 and 3.9 artifacts for Musl and Mac OS?

Reply:
No, those are not ordered by version for some reason, I re-ordered and moved them to above.